plugins/user-authenticators/ldap/src/org/apache/cloudstack/api/response/LdapConfigurationResponse.java

```diff
@@ -31,18 +34,27 @@
     @Param(description = "port")
     private int port;
 
+    @SerializedName("domain_id")
```

Comment:
This can be `domainid` like used/returned by other responses.

Reply:
here again, i see mixed conventions

Reply:
Yes, no strong opinions, just saying most API responses use domainid as response key name.

Reply:
@DaanHoogland where do you see mixed convention (on the API)? I can see the `domain_id` when it's refering a database column, but not in the API command/responses so far.

Reply:
ok, I think I may have mixed those up. tnx @marcaurele